packages/login/src/appFactory.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {combineReducers} from 'redux'
+import {Provider} from 'react-redux'
+import {StoreFactory, ExternalEvents, Intl} from 'tocco-util'
+import {addLocaleData} from 'react-intl'
+import {IntlProvider, intlReducer} from 'react-intl-redux'
+import {LoadMask} from 'tocco-ui'
+
+import LoginContainer from './containers/LoginContainer'
+import PasswordUpdateDialog from './containers/PasswordUpdateDialogContainer'
+import passwordUpdateReducers from './modules/passwordUpdate/reducers'
+import loginReducers, {sagas} from './modules/reducers'
+
+import {setUsername} from './modules/login/actions'
+import * as passwordUpdate from './modules/passwordUpdate/dialog/actions'
+
+import de from 'react-intl/locale-data/de'
+import en from 'react-intl/locale-data/en'
+import fr from 'react-intl/locale-data/fr'
+import it from 'react-intl/locale-data/it'
+
+export const loginFactory = (id, input, externalEvents) => {
+  var showTitle = input ? input.showTitle : false
+
+  var content = <LoginContainer showTitle={showTitle}/>
+  var dispatches = [
+    passwordUpdate.setShowOldPasswordField(false),
+    passwordUpdate.setForcedUpdate(true),
+    passwordUpdate.setStandalone(false)
+  ]
+
+  return factory(content, 'login', loginReducers, id, input, externalEvents, dispatches)
+}
+
+export const passwordUpdateFactory = (id, input, externalEvents) => {
```

Comment:
maybe we should check if mandatory input paramter `username` is available and throw or log an error if this is not the case?